docs/developer-docs/smart-contracts/overview/canister-lifecycle.mdx

```diff
@@ -0,0 +1,63 @@
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+
+# Canister lifecycle
+
+<MarkdownChipRow labels={["Beginner", "Concept"]} />
+
+## Write
+
+The ICP accepts and executes canister code in the WebAssembly (Wasm) binary format.
+Usually, developers write code in a higher-level language and then compile the code into a Wasm binary that can be deployed into a canister.
+
+## Deploy
+
+### Create
+
+A developer can create an empty canister on ICP by calling the system canister.
+It can be done using the `NNS frontend dapp` in the browser or `dfx` in the command line.
+The developer can specify the initial canister settings and choose the target subnet.
+Once the canister is created, the developer gets its canister ID and becomes the controller of the canister.
+
+<div class="text--center">
+<img src="/img/docs/create-canister.png" alt="Create canister" width="600"/>
+</div>
+
+### Install
+
+The next step after writing the code and creating a canister is installing the code.
+This can be done by calling the `install_code` endpoint of the system canister and passing the canister ID and the Wasm binary to it.
+The `dfx` tool has a command for making this call.
+
+<div class="text--center">
+<img src="/img/docs/install-canister.png" alt="Install canister" width="600"/>
+</div>
+
+## Call
+
+After installation, the canister accepts and processes messages (calls) from users and other canisters.
+Only the public endpoints of the canister can be called.
+These endpoints correspond to exported Wasm functions that are annotated with `canister_update` or `canister_query` in their names.
+
+<div class="text--center">
+<img src="/img/docs/call-canister.png" alt="Call canister" width="600"/>
+</div>
+
+
+## Maintain
+
+Certain system operations that change the state of the canister can be performed only by the controllers of the canister.
+The most important operation is upgrading the canister by installing different Wasm binary.
+In the initial stages of development, the canister is usually controlled by its developer, which allows the developer to iterate quickly.
+As the canister becomes mature and production ready, the control over it should move to a DAO in order to make the canister secure and trustworthy.
```

Comment:
```suggestion
As the canister becomes mature and production ready, the control over it could move to a DAO in order to make the canister secure and trustworthy.
```